### PR TITLE
Update RTC docs

### PR DESCRIPTION
**Pull request by bradfrizzell**

### CODE REVIEW

File: extensions/amp-a4a/rtc-documentation.md [hunk 1]
```diff
@@ -13,6 +13,8 @@ Remote HTML support was added for Delayed Fetch to support first-party cookie ta
 
 ## Overview
 
+RTC works by sending HTTP requests to URLs that a publisher specifies for each ad slot. The servers that receive these RTC requests respond with a JSON object that the associated AMP Fast Fetch implementation can use for various purposes, for instance the DoubleClick Fast Fetch implementation takes the data and adds it to the ad request to DFP as targeting parameters.
```
Comment (keithwrightbos) [inline extensions/amp-a4a/rtc-documentation.md]: sending XHR CORS requests... response with appropriate CORS headers and the body is a JSON object that...

File: extensions/amp-a4a/rtc-documentation.md [hunk 2]
```diff
@@ -24,9 +26,30 @@ The design of RTC is per-slot, with a** maximum of 5 parallel callouts allowed p
 
 In both cases, the results of these call-outs will be passed to the Fast Fetch implementations as part of ad url construction via the **getAdUrl** method. The ad network Fast Fetch implementation then uses results of these callouts to generate the ad URL. The semantics of how the ad network uses the RTC results to general the ad URL is specific to each individual network's implementation of Fast Fetch, so please refer to network-specific documentation for details.   
 
+## Glossary of Important Terms
+* **RTC Callout / RTC Request:** The HTTP request sent from the AMP page by the Fast Fetch implementation, to a server endpoint, asking for an RTC Response.
```
Comment (keithwrightbos) [inline extensions/amp-a4a/rtc-documentation.md]: XHR CORS request

File: extensions/amp-a4a/rtc-documentation.md [hunk 3]
```diff
@@ -24,9 +26,30 @@ The design of RTC is per-slot, with a** maximum of 5 parallel callouts allowed p
...
+
+* **RTC Response:** The JSON object returned in response to the RTC Callout.
```
Comment (keithwrightbos) [inline extensions/amp-a4a/rtc-documentation.md]: Ditto about response headers here

File: extensions/amp-a4a/rtc-documentation.md [hunk 4]
```diff
@@ -144,7 +167,7 @@ RTC requests are only sent to HTTPS endpoints.
 
 ### RTC Callout Endpoint and Response Specification
 
-The RTC endpoint must respond to the RTC GET request with a valid JSON object of targeting information to insert into the ad request URL. Redirects are not allowed, and if the endpoint attempts to redirect, the RTC will be abandoned. Special care should be made to respond to these requests as quickly as possible, as there is a 1 second default timeout imposed by RTC, and publishers may optionally shorten this timeout. Responses that return after the timeout will be dropped.
+The RTC endpoint must respond to the RTC GET request with a valid JSON object to insert into the ad request URL. Redirects are not allowed, and if the endpoint attempts to redirect, the RTC will be abandoned. Special care should be made to respond to these requests as quickly as possible, as there is a 1 second default timeout imposed by RTC, and publishers may optionally shorten this timeout. Responses that return after the timeout will be dropped.
```
Comment (keithwrightbos) [inline extensions/amp-a4a/rtc-documentation.md]: Same here... RTC XHR CORS request with CORS headers and...

**Comment by aghassemi** [comment]
Thanks for the Pull Request. Since this PR has been stale for a while, I am closing it. Please feel free to reopen as needed.

**Pull request closed without merging**